src/main/java/owen/Storage.java

```diff
@@ -0,0 +1,71 @@
+package owen;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import owen.task.TaskList;
+
+/**
+ * Storage handles reading and writing task list to file on disk.
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Create storage backed by a file on disk.
+     * @param path Path to file on disk.
+     */
+    public Storage(String path) {
+        this.file = new File(path);
+    }
+
+    /**
+     * Read TaskList from disk.
+     * @return TaskList read from disk.
+     */
+    public TaskList readTaskList() {
+        try {
+            TaskList taskList = new TaskList();
+
+            Scanner scanner = new Scanner(this.file);
+            while (scanner.hasNextLine()) {
+                String taskString = scanner.nextLine();
+                try {
+                    taskList = taskList.deserializeTask(taskString);
+                } catch (OwenException exception) {
+                    System.err.println(exception.getMessage());
+                }
+            }
+            scanner.close();
+
+            return taskList;
+        } catch (FileNotFoundException exception) {
+            return new TaskList();
+        }
+    }
+
+    /**
+     * Write TaskList to disk.
```

Comment:
Method header comments should start in the form "Writes"
https://se-education.org/guides/conventions/java/intermediate.html#comments